src/madetomeasure_api/routes/subscribers.clj

```diff
@@ -29,8 +29,8 @@
 (defn create-subscribers [request]
   (let [subscribers (extract-subscribers request)]
     (try
```

Comment:
Would it make sense to wrap some of these try/catch blocks in a separate function?  I assume we'll be adding a few more of these.